script.js

```diff
@@ -18,4 +19,194 @@ menu.addEventListener('click', () => {
-});
\ No newline at end of file
+});
+
+const projects = [
+  {
+    name: 'Tribute Page Practice',
+    description:
+      'This project I did in the collaborative sessions, but only the skeleton and other stuffs. A few days ago, I improve it, using css.',
+    projectImg: './images/tributepage.jpg',
+    techologies: [
+      'HTML',
+      'CSS',
+      'JavaScript',
+      'GitHub',
+      'Codepen',
+    ],
+    liveVersion: 'https://codepen.io/oliverscz/pen/WNOqWJw',
+    gitHubRepo: 'https://codepen.io/oliverscz/pen/WNOqWJw',
+  },
+  {
+    name: 'Landing Page Practice',
+    description:
+      'This project I did in the collaborative sessions, but only the skeleton and other stuffs. A few days ago, I improve it, using css',
+    projectImg: './images/landingpage.jpg',
+    techologies: [
+      'HTML',
+      'CSS',
+      'JavaScript',
+      'GitHub',
+      'Codepen',
+    ],
+    liveVersion: 'https://codepen.io/oliverscz/pen/NWgQwPY',
+    gitHubRepo: 'https://codepen.io/oliverscz/pen/NWgQwPY',
+  },
+  {
+    name: 'Survey Form Practice',
+    description:
+      'This project I did in the collaborative sessions, but only the skeleton and other stuffs. A few days ago, I improve it, using css.',
+    projectImg: './images/surveyform.jpg',
+    techologies: [
+      'HTML',
+      'CSS',
+      'JavaScript',
+      'GitHub',
+      'GitFlow',
+    ],
+    liveVersion: 'https://codepen.io/oliverscz/pen/zYdOrxB',
+    gitHubRepo: 'https://codepen.io/oliverscz/pen/zYdOrxB',
+  },
+  {
+    name: 'Portfolio Mobile Webpage',
+    description:
+      'I am continuously working on this project, as I receive training and new knowledge, I come here and add all that to this project.',
+    projectImg: './images/portfolio-project.jpg',
+    techologies: ['HTML', 'CSS', 'JavaScript', 'GitHub', 'VScode'],
+    liveVersion: 'https://github.com/oliverSCZ/mobile-version-project',
+    gitHubRepo: 'https://github.com/oliverSCZ/mobile-version-project',
+  },
+  {
+    name: 'Portfolio Mobile Webpage',
+    description:
+      'I am continuously working on this project, as I receive training and new knowledge, I come here and add all that to this project.',
+    projectImg: './images/portfolio-project.jpg',
+    techologies: ['HTML', 'CSS', 'JavaScript', 'GitHub', 'VScode'],
+    liveVersion: 'https://github.com/oliverSCZ/mobile-version-project',
+    gitHubRepo: 'https://github.com/oliverSCZ/mobile-version-project',
+  },
+  {
+    name: 'Portfolio Mobile Webpage',
+    description:
+      'I am continuously working on this project, as I receive training and new knowledge, I come here and add all that to this project.',
+    projectImg: './images/portfolio-project.jpg',
+    techologies: ['HTML', 'CSS', 'JavaScript', 'GitHub', 'VScode'],
+    liveVersion: 'https://github.com/oliverSCZ/mobile-version-project',
+    gitHubRepo: 'https://github.com/oliverSCZ/mobile-version-project',
+  },
+];
+
+const loadProject = (projects) => {
```

Comment:
**Kindly follow the design guidelines requested on your Figma template:**

_**Template:**_ 

<img width="1440" alt="Screen Shot 2021-11-19 at 5 14 03" src="https://user-images.githubusercontent.com/22258274/142613695-99bbef0f-ad6d-4d16-bc04-6de587fb3c9c.png">


#
_**Yours:**_ 
<img width="1440" alt="Screen Shot 2021-11-19 at 5 13 32" src="https://user-images.githubusercontent.com/22258274/142613618-aa6ae8df-d86a-4f88-b85f-097ad72ee686.png">

- To improve: Fonts, margins, paddings, your buttons etc.

Reply:
Thank you for all your advices Mr. Carlos @carlos-ssh  and Mr. Arthur @ArthurGC , that helps me to keep improving. 👍